libs/design/src/molecules/paginator/paginator.component.html

```diff
@@ -0,0 +1,21 @@
+<nav role="navigation" attr.aria-label="Pagination Navigation">
```

Comment:
The `nav` can be removed since `daff-paginator` can hold the `role`.

Reply:
done